src-tauri/Cargo.toml

```diff
@@ -66,19 +66,22 @@ features = [
 winreg = "0.10.1"
 
 [dependencies.tauri]
-features = ["clipboard-all", "dialog-all", "fs-all", "global-shortcut-all", "gtk-tray", "notification-all", "os-all", "path-all", "process-exit", "process-relaunch", "shell-all", "system-tray", "window-all", "updater"]
+features = ["clipboard-all", "dialog-all", "fs-all", "global-shortcut-all", "gtk-tray", "notification-all", "os-all", "path-all", "process-exit", "process-relaunch", "shell-all", "system-tray", "updater", "window-all"]
 version = "1.0.0-rc.11"
 
 [features]
 default = [ "custom-protocol" ]
 custom-protocol = [ "tauri/custom-protocol" ]
+opencl = ["subspace-farmer/opencl"]
 
 [profile.dev]
 opt-level = 3
 
+[profile.release]
```

Comment:
Why `release` and not `production`? It might be still desirable to build `release` in some cases while not spending quite as much time as with these parameters.